tests/model_fields/tests.py

```diff
@@ -698,16 +706,30 @@ class BigIntegerFieldTests(IntegerFieldTests):
     documented_range = (-9223372036854775808, 9223372036854775807)
 
 
+@unittest.skipIf(connection.vendor == 'mysql', "Test not for MySQL")
```

Comment:
Good catch, it caused me to double check SQLite. I had an incorrect assumption that it was not using unsigned integers.